From acdf73117af8f0bc9ee92e83da282a87f96230f7 Mon Sep 17 00:00:00 2001
From: Nikhil Devshatwar <nikhil.nd@ti.com>
Date: Tue, 29 Sep 2015 19:51:05 +0530
Subject: [PATCH 07/12] arm: dts: dra72: infoadas: Delete conflicting gpio hog

To reroute the ethernet on J6eco, the board GPIO is hogged LOW.
But for the LVDS use case, vin4b is to be selected over ethernet.
When the sensor driver requests this GPIO, it fails because of the hog.

Delete the hog node in infoadas-LVDS dtb so that the 4 cameras can be
enabled.

Change-Id: I6bfa5a68ed1cb6c501a7a8b1675a4376b8341d60
Tested-by: Chetan Matad <x0199489@ti.com>
Signed-off-by: Nikhil Devshatwar <nikhil.nd@ti.com>
---
 arch/arm/boot/dts/dra72-evm-infoadas-LVDS.dts |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/dra72-evm-infoadas-LVDS.dts b/arch/arm/boot/dts/dra72-evm-infoadas-LVDS.dts
index f794739..2c765c5 100644
--- a/arch/arm/boot/dts/dra72-evm-infoadas-LVDS.dts
+++ b/arch/arm/boot/dts/dra72-evm-infoadas-LVDS.dts
@@ -27,6 +27,7 @@
 	export-devnode;
 };
 
+/delete-node/ &sel_enet_mux_hog;
 /delete-node/ &wdt2;
 /delete-node/ &vpe;
 
-- 
1.7.9.5

